easybuild/easyconfigs/g/gofasta/gofasta-0.0.5.eb

```diff
@@ -0,0 +1,26 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'GoPackage'
+
+name = 'gofasta'
+version = '0.0.5'
+
+homepage = 'https://github.com/cov-ert/gofasta'
+description = """Some functions for dealing with alignments, 
+developed to handle SARS-CoV-2 data as part of the COG-UK project."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/cov-ert/gofasta/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['2d66f1bfd68cc673b4991634eedda14faf5f8c3e6ca578b18d68b3763a45606b']
+
+builddependencies = [
+    ('Go', '1.16.5'),
+]
+
+# options = {'modulename': 'absl'}
```

Comment:
?

Reply:
I really need new glasses. Thanks for spotting this.